src/langs/en/messages.json

```
@@ -71,5 +71,26 @@
   "IMPORT_ZIP": "The ZIP archive was successfully imported",
   "IMPORT_H5P": "The H5P package was successfully imported",
   "DELETE_MEMBER": "The account was successfully deleted",
-  "UPDATE_PASSWORD": "The password was updated successfully"
+  "UPDATE_PASSWORD": "The password was updated successfully",
+
+  "INVALID_ARCHIVE_FILE": "the submitted file is not a zip archive",
+  "INVALID_FILE_ITEM": "The file's properties are invalid",
+  "UNEXPECTED_EXPORT_ERROR": "An error occured while exporting the zip file",
+
+  "NOT_APP_ITEM": "The targeted item is not an application item",
+  "INVALID_APP_ORIGIN": "The application does not match its registered origin is invalid",
+  "TOKEN_ITEM_ID_MISMATCH": "The auth token does not match the targeted item",
+  "APP_DATA_NOT_FOUND": "No app data were not found",
```

There's adouble negation here
```suggestion
  "APP_DATA_NOT_FOUND": "No app data were found",
```